app/pages/summary/contextCreator.test.js

```diff
@@ -462,6 +463,7 @@ describe('order summary contextCreator', () => {
         serviceRecipientsOdsCode: 'A10001',
         cataloguePriceType: 'Flat',
         catalogueItemName: 'Some item name',
+        serviceItemId: 'service-item-id',
```

Comment:
`serviceInstanceId` and `service-instance-id` to be consistent with the business jargon?